src/container/srv_target.c

```diff
@@ -843,7 +843,10 @@ cont_child_started(struct ds_cont_child *cont_child)
 static void
 cont_child_stop(struct ds_cont_child *cont_child)
 {
-	if (!cont_child->sc_stopping) {
+	/* Some ds_cont_child will only created by ds_cont_child_lookup().
+	 * never be started at all
+	 */
+	if (!cont_child->sc_stopping && cont_child_started(cont_child)) {
```

Comment:
Could you explain in detail why "Some ds_cont_child will only created by ds_cont_child_lookup()"? The stacktrace in DAOS-9314 is in a enumeration handler, which means the container must be opened, I don't see how can it not being started.

Maybe we'd add a D_ASSERT(cont_child_started(cont_child)) instead?